Automattic/wp-calypso
# Plugins: Refactor Education Footer

## Proposed Changes

I'm curious on if others will agree this is a worthwhile change! I think that the Plugins Education footer can be refactored quite heavily. It's seemingly the only thing which relies on a `LinkCard` component, but that can be very easily recreated (it seems that the current component tries to redo a lot of styles unnecessary). 

I'm not that familiar with TypeScript, so I'd love to hear if there's a specific reason for the original changes that I missed.

## Testing Instructions

Go to `/plugins` and verify that these two are the same. 

<img width="1357" alt="Screenshot 2024-02-14 at 13 48 24" src="https://github.com/Automattic/wp-calypso/assets/43215253/f51b0c24-47c6-4d0d-96db-d0e68af85ad3">

<img width="1104" alt="Screenshot 2024-02-14 at 13 51 39" src="https://github.com/Automattic/wp-calypso/assets/43215253/311a02f3-aaf3-41ac-9d7b-bd3a9a9f9653">
 
cc @tyxla
